src/utils/paths.js

```diff
@@ -18,6 +19,7 @@ function ensureDir(dir) {
 }
 
 module.exports = {
-  resolvePath,
   ensureDir,
+  python2: which.sync('python2') || which.sync('python'),
```

Comment:
given that this will not _always_ be python2, we should probably name this just `python`

Reply:
Alright, FWIW my thought was that since we're currently explicitly requiring python2, it's marginally useful to make that explicit in the variable name, and that the variable name could be changed when that dependency changes. I don't feel strongly about either approach though :slightly_smiling_face: 

Changed in 207c8c9b0e55d3753c5d07169d984e7b71778342